issue_manager_script.py

```diff
@@ -96,27 +62,26 @@
 try:
     access_token = integration.get_access_token(installation_id).token
 except GithubException as e:
-    print(f"Failed to get access token: {e}")
+    print(f"Failed to get access token: {e.data}")
     exit(1)
 
 # Initialize a GitHub client using the access token
 g = Github(login_or_token=access_token)
 
-# Define variables for your project
-project_number = 3  # Replace with your project's number
+# ----------------------- Fetch Organization and Project ----------------------
 
 # Fetch the organization object
 try:
     org = g.get_organization(org_name)
 except GithubException as e:
-    print(f"Failed to get organization {org_name}: {e}")
+    print(f"Failed to get organization {org_name}: {e.data}")
```

Comment:
I love all these nuanced logging features! Is there somewhere new DUNA members can go to review deployment runs? 🧑‍💻

Reply:
These logs are available in GH actions; I don't believe they are front-facing, so members may not be able to see them. Perhaps we can discuss if this is something we should change (or if it's a security risk), how to implement this, and who would have access to it?

Reply:
We love transparency here! Certainly if there is any personal data involved in this kind of logging script, then we might need to reconsider, but if it can be a source for any number of potentially public data [changes to structure, token distribution, new proposals, support metrics, deploying updates] then maybe we could find a way to create a public mirror. Would an eventual UI find it useful to pull data from this log? 

If it could track DUNA participation then we'd also need to see how it

Reply:
@tehchives It's moreso technical/command line logs that I'm worried about here with unnecessarily exposing potential vulnerabilities or attack vectors. [Here](https://github.com/WhyDRS/Database/actions/runs/11866131493/job/33072413963) Is an example of a job that was run where you can uncollapse each action that was performed (below is what an external user would see; they can't expand on each line item). Would it help DAO members if these logs were publicly available?

![Screenshot 2024-11-16 at 12 22 12 PM](https://github.com/user-attachments/assets/7eab2c00-5819-41b6-ae34-1a7d5ed9156e)

Reply:
This looks pretty nuanced, and probably a little more geared towards the most active organization members here with write access. Consolidarily, the preview logs for every public site deploy are available without login through the chat app from Fleek or Netlify. Closing this out for now given the successful merge, and can always reference again in a later issue. :)